ccd-definition/CaseEventToFields/DefendantResponse.json

```diff
@@ -8,108 +8,189 @@
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
     "FieldShowCondition": "respondent1ClaimResponseType = \"DO_NOT_SHOW_IN_UI\"",
-    "PageID": "RespondentResponseType",
+    "PageID": "ConfirmDetails",
     "ShowSummaryChangeOption": "N"
   },
   {
     "CaseTypeID": "CIVIL",
     "CaseEventID": "DEFENDANT_RESPONSE",
-    "CaseFieldID": "respondent1ClaimResponseType",
-    "PageDisplayOrder": 2,
+    "CaseFieldID": "respondent2Copy",
+    "PageDisplayOrder": 1,
     "PageFieldDisplayOrder": 1,
-    "DisplayContext": "MANDATORY",
-    "PageID": "RespondentResponseType",
-    "ShowSummaryChangeOption": "Y"
+    "DisplayContext": "READONLY",
+    "FieldShowCondition": "respondent1ClaimResponseType = \"DO_NOT_SHOW_IN_UI\"",
```

Comment:
Should it be respondent2ClaimResponseType?

Reply:
no, this is a CCD hack to permenantly hide a field. It can be any field